src/display/annotation_layer.js

```diff
@@ -239,6 +239,40 @@ class AnnotationElement {
     return container;
   }
 
+  /**
+   * Create areas for the quadPoints.
+   *
+   * @private
```

Comment:
After this line add: `@param {boolean} ignoreBorder` (just like `_createContainer`).